### PR TITLE
[OBSDEF-1200]Add prvg rsyslog support

### PR DESCRIPTION
**Pull request by CraneShiEMC**
## Purpose
[OBSDEF-1200](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-1200)
_List the changes for this PR_
add prvg/logback.xml, add labels for segmentstore, controller pods, add relavant crd fields
## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [x] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/stream-master-ci/58/parameters/

### CODE REVIEW

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
Crane,  it seems you branch is out of date.  can you run this in your branch:

```
git pull origin master 
``` 
some of the files are already in master.

**Comment by CraneShiEMC** [comment]
> Crane, it seems you branch is out of date. can you run this in your branch:
> 
> ```
> git pull origin master 
> ```
> 
> some of the files are already in master.

@eddiepavkovic Now, I have merged latest master commits.

File: ecs-cluster/templates/ecs-cluster.yaml [hunk 1]
```diff
@@ -208,12 +208,16 @@ spec:
...
+    {{- if .Values.bookkeeper.image.tag }}
+    version: {{ .Values.bookkeeper.image.tag }}
     {{- else }}
     version: {{ .Values.tag }}
     {{- end }}
     blockOwnerDeletion:  {{ .Values.bookkeeper.blockOwnerDeletion }}
+    labels:
+      app.kubernetes.io/namespace: {{ .Release.Namespace }}
+      objectscale.dellemc.com/logging-inject: "true"
+      objectscale.dellemc.com/logging-release-name: "{{ .Release.Name }}"
```
Comment (eddiepavkovic) [inline ecs-cluster/templates/ecs-cluster.yaml:220]: @vazhem - should this be a macro from `common-libs`?

File: zookeeper-operator/values.yaml [hunk 2]
```diff
@@ -25,5 +25,5 @@ replicaCount: 1
 # The image configured for the Zookeeper Operator
 image:
   repository: zookeeper-operator
-  tag: 2.10.0-b1519ae # rfw-update-this zookeeper-operator-docker-image
+  tag: 0.2.9-173-5c86f53 # rfw-update-this zookeeper-operator-docker-image
```
Comment (eddiepavkovic) [inline zookeeper-operator/values.yaml:28]: this should be `0.2.10...`

**Review by eddiepavkovic — APPROVED** [review]
lgmt

**Review by realAaronWu — APPROVED** [review]

File: ecs-cluster/templates/ecs-cluster.yaml (hunk 1 above)
Comment (vazhem) [reply ecs-cluster/templates/ecs-cluster.yaml:220]: add dependency on monitoring lib and use
```
{{ include "common-monitoring-lib.logging-inject-labels" . | indent 8 }}
```
(with correct indentation)

Comment (fedor-chemashkin) [reply ecs-cluster/templates/ecs-cluster.yaml:220]: We shouldn't put these labels here

**Pull request merged**